knn-banking-product-recommender/deployment/gradio.requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Python 3.12
+gradio>=4.31.4
+highwind==1.0.0
```

Comment:
Please can you bump this to `1.1.1`? That version has the fixes for the refresh token.